.github/workflows/tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 
       - name: Archive Screenshot Results on Error
         if: failure()
-        uses: actions/upload-artifact@v1
+        uses: actions/upload-artifact@v3
```

Comment:
Strange that Dependabot did not propose to upgrade this.

Reply:
not sure where I copied it from :facepalm:  it should fix the missing artifacts in https://github.com/vector-im/element-android/pull/7137 (which now correctly fails) :crossed_fingers: